### PR TITLE
Rent a new linter

### PR DESCRIPTION
**Pull request by rafaellehmkuhl**
Uses vue/recommended + vue/airbnb rules on Eslint

I did the necessary changes on the main frontend files to comply to the new rules.
There are several changes needed on the old/dev frontend that I still need to make, but I'm already opening the PR so you can review the structure.

### CODE REVIEW

**Review by patrickelectric — CHANGES_REQUESTED** [review]
@rafaellehmkuhl it appears that CI is failing.

**Comment by rafaellehmkuhl** [comment]
> @rafaellehmkuhl it appears that CI is failing.

Yes. It's failing on the development frontend. I'm finishing the fix on those.

**Comment by rafaellehmkuhl** [comment]
@patrickelectric just finished those. I will test the frontends to see if nothing was broken during the lint fixes.

**Review by patrickelectric — APPROVED** [review]

**Comment by rafaellehmkuhl** [comment]
Just tested the frontends. There was a small bug I had introduced on the versionchooser interface, making it not show the entire repo name up, but its now fixed.

**Review by Williangalvani — APPROVED** [review]

File: core/services/versionchooser/frontend/index.html [hunk 1]
```diff
@@ -156,10 +156,18 @@ <h3>Applying...</h3>
   </body>
 
   <script>
-    /*global Metro axios, Vue, timeago*/
-    new Vue({
+    /* global Metro axios, Vue, timeago */
+    new Vue({ // eslint-disable-line no-new
       el: '#app',
-      data () {
+      filters: {
```
[the remark this answers is not in the document]
Comment (rafaellehmkuhl) [reply core/services/versionchooser/frontend/index.html:162]: [This one](https://eslint.vuejs.org/rules/order-in-components.html)

**Pull request merged**